shared/login/register/paper-key/index.render.desktop.js

```diff
@@ -17,6 +17,7 @@ const Render = ({onBack, onSubmit, onChangePaperKey, error, paperKey, waitingFor
         rowsMax={3}
         style={styles.input}
         errorText={error}
+        floatingHintTextOverride='Paper key'
```

Comment:
We use "Paper key" or "paper key" as 2 words everywhere we show this text to humans.